src/diffusers/models/resnet.py

```diff
@@ -640,6 +640,229 @@ def forward(self, input_tensor, temb):
         return output_tensor
 
 
+class Mish(torch.nn.Module):
```

Comment:
Could we add this to the activations.py function? See here: https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/activations.py -> Also there is already a Mish function, can't we just use this one?

Reply:
Yes, Let me change that and test it out.

Reply:
Works with the existing one. Replaced with `get_activation` for now.